jgclark.NoteHelpers/plugin.json

```diff
@@ -49,6 +49,14 @@
       "description": "Look for Links to headings and make them work by converting them to plugin command calls",
       "jsFunction": "convertLocalLinksToPluginLinks"
     },
+    {
+      "name": "find & link unlinked notes",
```

Comment:
As this could be used from an x-callback, I suggest changing '&' to 'and', to reduce possible confusion.